src/main/java/org/mifos/processor/bulk/camel/routes/ProcessorStartRoute.java

```diff
@@ -194,7 +194,9 @@ private void setup() {
                     JSONObject response = new JSONObject();
 
                     try {
+                        logger.info("Tenant name {}", exchange.getProperty(TENANT_NAME).toString());
```

Comment:
additional log statements for debug should be of debug level

Reply:
Ok